pom.xml

```diff
@@ -67,7 +67,7 @@
   <properties>
     <revision>2.6.2</revision>
     <changelist>-SNAPSHOT</changelist>
-    <jenkins.version>2.222.4</jenkins.version>
+    <jenkins.version>2.249.1</jenkins.version>
```

Comment:
jcasc baseline

Reply:
Due to #230, this will be 2.308 ;)

Reply:
aware but depends which is merged first, trivial conflict resolution

Reply:
ah i see merged